src/Router.tsx

```diff
@@ -0,0 +1,59 @@
+import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
+
+import { ProtectedRoute } from './components';
+import { ROUTES } from './routes/Routes';
+import {
+    BaseScreen,
+    Chat,
+    EmailConfirmation,
+    Feed,
+    Home,
+    Login,
+    NotFound,
+    Post,
+    Profile,
+    Register,
+    ResetPassword,
+    Search
+} from './screens';
+
+export const Router = () => {
+    return (
+        <BrowserRouter>
+            <Routes>
+                <Route path={ROUTES.HOME} element={<Home />} />
+                <Route path={ROUTES.NOT_FOUND} element={<NotFound />} />
+                <Route
```

Comment:
what's the difference between error and not_found route? I think that we need only not_found

Reply:
but I think it would be prettier if every route that we don't have will display on /not-found